scripts/up.sh

```diff
@@ -12,6 +12,17 @@ set -a
 source "$ENV_FILE"
 set +a
 
+# TODO: allow more than just --verbose env var overrides
+while (( "$#" )); do
+  case "$1" in
+    -v|--verbose)
+      VERBOSE_HARNESS=1
+      ;;
+  esac
+  shift
+done
```

Comment:
so I assume that, this chunk of shell actually can read the whole arg option, and find `-v/--verbose` and assign `VERBOSE_HARNESS` to be 1, right? looks neat.

Reply:
thx - but just copy pasta from other scripts we have